tests/integration/new-test-helpers-in-new-integration-tests-test.js

```diff
@@ -16,6 +16,15 @@ module('Integration | {{t}} Helper', function (hooks) {
     assert.equal(t("pluralized.translation", { count: 1 }), "One Click", "test-helpers t returns translation");
     assert.equal(this.element.querySelector('#t-target').textContent, t("pluralized.translation", { count: 1 }));
   });
+
+  test("expectTranslation test helper", async function (assert) {
+    assert.expect(1);
+    await render(hbs`
```

Comment:
TIL `render` is async

https://github.com/emberjs/ember-test-helpers/blob/767d54d444996587633a54959f2c88d00cc76df4/addon-test-support/%40ember/test-helpers/setup-rendering-context.js#L58